wire-ios-sync-engine/Source/Synchronization/Strategies/TeamDownloadRequestStrategy.swift

```diff
@@ -31,17 +33,17 @@ struct TeamPayload: Decodable {
     let identifier: UUID
     let name: String
     let creator: UUID
-    let binding: Bool
     let icon: String
     let iconKey: String?
+    let splashScreen: String?
 
     private enum CodingKeys: String, CodingKey {
         case identifier = "id"
         case name
         case creator
-        case binding
```

Comment:
According to swagger: 

binding is deprecated, and should be ignored. The non-binding teams API is not used (and will not be supported from API version V4 onwards), and binding will always be true.